test/extended/cli/basics.go

```diff
@@ -198,9 +198,22 @@ var _ = g.Describe("[sig-cli] oc basics", func() {
 			ok := rootCAs.AppendCertsFromPEM(kubeCA)
 			o.Expect(ok).To(o.BeTrue())
 
+			// Some deployment types (such as MicroShift) dont have
+			// OAuth tokens, therefore we need to configure
+			// regular certificates instead.
+			var certs []tls.Certificate
+			certData := oc.UserConfig().CertData
+			keyData := oc.UserConfig().KeyData
+			if len(certData) > 0 && len(keyData) > 0 {
+				cert, err := tls.X509KeyPair(certData, keyData)
+				o.Expect(err).NotTo(o.HaveOccurred())
+				certs = []tls.Certificate{cert}
+			}
+
 			transport = &http.Transport{
 				TLSClientConfig: &tls.Config{
-					RootCAs: rootCAs,
+					RootCAs:      rootCAs,
+					Certificates: certs,
```

Comment:
@pacevedom does this change automatically implies OAuth tokens will no longer be used? Is there any way to detect whether OAuth tokens are used and generate list of certs only when they are not?

Reply:
Thanks for having a look, this was a leftover from before all the changes to the oc project impersionation were complete. Now its not needed so I remove it.